src/pages/markets/_markets-style.js

```diff
@@ -57,27 +57,18 @@ export const MarketsList = styled(CssGrid)`
     border-left: 1px solid var(--color-grey-22);
     border-right: ${({ has_right_border }) =>
         has_right_border ? '1px solid var(--color-grey-22)' : 'unset'};
-    grid-template-columns: ${({ col }) => `repeat(${col ?? 5}, 1fr)`};
+    grid-template-columns: ${({ responsive_col, col }) =>
+        `repeat(${responsive_col + 2 || col || 3}, 1fr)`};
```

Comment:
By having this instead of the previous implementation, will it not limit the flexibility of the column implementation in the future? For e.g. if I require a market list column for desktop|tablet|mobile of 4|3|1 or 5|3|1

Reply:
I don't think that there will be a case where desktop is 5 as 4 is already too many.
I can't think of a situation where tablet is 3 and mobile is 1, because if tablet have a room for 3 columns, mobile will have a room for 2. 

We can specify a flag to each one of them `desktop-tablet-mobile` to simplify the implementation a bit.